omnigibson/prims/joint_prim.py

```diff
@@ -169,9 +169,11 @@ def _initialize(self):
                 dof_props = self._dc.get_dof_properties(dof_handle)
                 self._dof_handles.append(dof_handle)
                 self._dof_properties.append(dof_props)
-                # Infer control type based on whether kp and kd are 0 or not
+                # Infer control type based on whether kp and kd are 0 or not, as well as whether this joint is driven or not
                 kp, kd = dof_props.stiffness, dof_props.damping
-                if kp == 0.0:
+                if not self._driven:
```

Comment:
on a high-level, when would `driven` be False for a joint? What type of joint is this?

Reply:
driven = False for joints that shouldn't have a motor attached (e.g.: furniture joints). Realistically the only joints that **should** be driven are robot joints.

In general, I'm going to take a closer look at this distinction as soon as @cgokmen is finished refactoring the physics articulation tensor API. Our hunch is that this distinction is also partly causing issues with tiago.